### PR TITLE
Remove hard requirement on shopify_api ~>3.2.0

### PR DESCRIPTION
**Pull request by stephenminded**
And bump gem to `6.0.5`

@kevinhughes27 @shawnfrench

### CODE REVIEW

File: shopify_app.gemspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency('rails', '>= 3.1', '< 5.0')
 
-  s.add_runtime_dependency('shopify_api', '~> 3.2.0')
+  s.add_runtime_dependency('shopify_api', '> 3.2.0')
   s.add_runtime_dependency('omniauth-shopify-oauth2', '~> 1.1.4')
```
Comment (kevinhughes27) [inline shopify_app.gemspec]: can we also remove the hard req here

File: shopify_app.gemspec [hunk 2]
```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency('rails', '>= 3.1', '< 5.0')
 
-  s.add_runtime_dependency('shopify_api', '~> 3.2.0')
+  s.add_runtime_dependency('shopify_api', '> 3.2.0')
```
Comment (shawnfrench) [inline shopify_app.gemspec]: Should this be `>=`?

Comment (stephenminded) [inline shopify_app.gemspec]: Yeah, strictly speaking that's probably the best way to specify it. I'll change it

**Comment by stephenminded** [comment]
OK, updated... another set of :eyes: ?

**Pull request closed without merging**